src/app/services/file-upload.service.ts

```diff
@@ -62,7 +62,11 @@ export class FileUploadService {
                 );
             }),
             catchError(() => {
-                throw new DatasetOperationError([new Error("File could not be loaded")]);
+                throw new FileUploadError([
+                    new Error(
+                        "File could not be loaded. Supported file types: CSV, JSON, Newline-delimited JSON, Geo JSON, Newline-delimited Geo JSON, ESRI Shapefile, Parquet",
+                    ),
+                ]);
```

Comment:
I do not see a check for HTTP response status 415

Reply:
fixed